bayesian/api/api_v2.py

```diff
@@ -202,6 +203,63 @@ def get(ecosystem, package, version):
 
         raise HTTPError(404, msg)
 
+    @staticmethod
+    def post():
+        """Handle the POST REST API call.
+
+        Component Analyses Batch is a 4 Step Process:
+        1. Gather and clean Request.
+        2. Query GraphDB Vendor Specific Edge
+        3. Build Known Package Response and Trigger componentApiFlow
+        4. Handle Unknown Packages and Trigger bayesianApiFlow
+        """
+        response_template = namedtuple("response_template", ["message", "status"])
+        ingestion_disabled_msg: str = "No data found for any package in manifest file. " \
+                                      "Ingestion flow skipped as DISABLE_UNKNOWN_PACKAGE_FLOW " \
+                                      "is enabled"
+        no_package_available_msg: str = "No Package in given manifest is available. " \
+                                        "Packages will be available shortly, " \
+                                        "Please retry after some time."
+
+        # Step1: Gather and clean Request
+        input_json: dict = request.get_json()
```

Comment:
Keep uniformity in type name. Use Dict all the places instead dict.

Reply:
To use `Dict` here, we have to do extra import which create an extra Overhead, and for this Specific purpose, I don't see it a worth.

Reply:
I can' really understand the overhead in importing a type., that is the norm across code who uses python typing. 

I also see that your other changes imports Dict. e.g. bayesian/utility/v2/ca_response_builder.py

Reply:
Yes, That's true. in `ca_response_builder` typing module is imported because we can do things like `Tuple[Dict]` means defining types of nested objects.

But in this particular case, `dict` should be suffice.

Reply:
What is the point of uniformity and usage recommendation? There won't be any measurable impact on importing a type. 
BTW, you are programming in Python, which is a worst language in terms of performance. You should be more focusing on consistency and readability than micro performance implication.

```suggestion
        input_json: Dict[ = request.get_json()
```

Reply:
> What is the point of uniformity and usage recommendation? There won't be any measurable impact on importing a type.
> BTW, you are programming in Python, which is a worst language in terms of performance. You should be more focusing on consistency and readability than micro performance implication.

Your Statement completely Contradicts your Previous Statement. 
https://github.com/fabric8-analytics/fabric8-analytics-server/pull/655#discussion_r465105468 

FYI: In Logging Discussion: You don't favor format consistency and showed concern about micro Performance.

Reply:
Do you see the point of consistency in logging? Almost 90% of the logging statement used here in API server follows `"msg %s"` format., why you want to differ from it? Isn't a python recommended way of doing it?

Reply:
> Your Statement completely Contradicts your Previous Statement.
> #655 (comment)
> 
> FYI: In Logging Discussion: You don't favor format consistency and showed concern about micro Performance.
Even your statement contradicts, you are so concerned about the performance of an import, but not ready to change the recommended logging approach.

Reply:
Let's consider Micro Performance too..
Ack this suggestion.
Ack Logging Suggestion.
Converted Log.info to log.debug.